routes/distribution.js

```diff
@@ -77,7 +61,7 @@ router.patch(
   async (req, res) => {
     const required = req.query.required;
     const id = req.query.id;
-    if (!required || !id) {
+    if (Number.parseInt(required) === NaN || !id) {
```

Comment:
PATCH api/distributions/updateRequiredCredits not used atm. 

could be useful in the future though? for customizable distributiosn